index.html

```diff
@@ -19,7 +19,47 @@
         color: red;
         font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
         font-size: 40px;
+        width: 75px;
+        height: 75px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
       }
+
+      .team-buttons {
+        position: relative;
+      }
+
+      button {
+        border-radius: 50%;
+        width: 70px;
+        height: 70px;
+        display: block;
+        background-color: lightgrey;
+        font-size: 20px;
+        font-weight: bold;
+        border: black 2px solid;
+      }
+
+      .button1 {
+        position:absolute;
+        top: 25px;
+        left: 25px;
+        
+      }
+
+      .button3 {
+        position:absolute;
+        top: 25px;
+        right: 25px;
+      }
+
+      .button-1 {
```

Comment:
the naming of button-1 is ambiguous. i think we should name these like button-one, button-three, button-minus-one.